airbyte-webapp/src/views/Connector/ServiceForm/components/Sections/ArraySection.tsx

```diff
@@ -1,33 +1,75 @@
 import { FieldArray, useField } from "formik";
-import React from "react";
+import React, { useMemo, useState } from "react";
 
 import { ArrayOfObjectsEditor } from "components";
 import GroupControls from "components/GroupControls";
 
-import { FormObjectArrayItem } from "core/form/types";
+import { FormBlock, FormGroupItem, FormObjectArrayItem } from "core/form/types";
 
-import { useServiceForm } from "../../serviceFormContext";
+import styles from "./ArraySection.module.scss";
 import { SectionContainer } from "./common";
-import { FormSection } from "./FormSection";
+import { VariableInputFieldForm } from "./VariableInputFieldForm";
 
 interface ArraySectionProps {
   formField: FormObjectArrayItem;
   path: string;
   disabled?: boolean;
 }
 
-/**
- * ArraySection is responsible for handling array of objects
- * @param formField
- * @param path
- * @constructor
- */
+const getItemName = (item: Record<string, string>): string => {
+  return Object.keys(item)
+    .sort()
+    .map((key) => `${key}: ${item[key]}`)
```

Comment:
Makes sense to add this change. My original intent was that the names were going to be too long and so it would show less but then it's more confusing that there's two names.